feat: adding Earthworm Jim channel

cpbellini opened the pull request:
### Description
Adds Earthworm Channel to the list of (awesome) GDQ Break Channels
This channel features Earthworm Jim shooting enemies from the first level of the game. Enemies are spawned by donations. If a donation is under $100 dollars, the enemy spawned is a crow, if it is over $100, the enemy is a Fifi. The channel number is 94, the year of the games first release.
![ezgif-4-546d056a30](https://github.com/GamesDoneQuick/gdq-break-channels/assets/47122702/942f616a-d323-43af-b737-9d0dee531883)


### Checklist:
- [x] I have read and followed the requirements in the [**Contributing** document](https://github.com/GamesDoneQuick/gdq-break-channels/blob/main/CONTRIBUTING.md).
- [x] My code has been tested.
- [x] My commit title follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) formatting.
- [x] All the code is my own, or is code I have the rights to, and is being made available under the [Apache License Version 2.0](https://www.apache.org/licenses/LICENSE-2.0).
- [x] My channel contains no first-party Nintendo assets.
- [x] I understand that GamesDoneQuick may reject, make changes to, or choose not to show this channel on broadcast, and that discussion or merging does not guarantee the work will be used.

VodBox commented:
Hi, sorry for taking a bit to get around to your submission. Thank you very much for submitting a channel!

I just have a few concerns that I'd like to see addressed before I take a review of the code itself.

1. I personally think the shadow colors ought to be different. The initial solid drop shadow ought to be darker and provide more contrast between it in and the text in front, and the brown blends a bits of the terrain and clashes with the purple of other parts. Having a shadow be brighter than some of the things it's in front of also feels slightly strange. Could this maybe just be a black?
2. When a lot of donations come in quick succession, the queue can get quite long, and especially with the time between monster spawns, it can take some time to get through it all. Generally it's good if the response to a donation can be more immediate or be capped in some way, that way it's very unambiguous what the donations are actually causing.
3. (Potentially also a solution to 2): Could this maybe be zoomed in so Earthworm Jim and the action take up more of the screen? Having so much of the screen be static but a single element off to one side always in motion draws attention both to that odd thing, but also the stillness of everything else. If it took up a larger portion of the frame, this feeling may be lessened. It would also give enemies a shorter path to reach, allowing for that queue to be emptied quicker.

cpbellini commented:
Updated to zoom in to shorten dono times, also changed tracker shadow colors.
![image](https://github.com/GamesDoneQuick/gdq-break-channels/assets/47122702/a3a94f64-2cba-4b84-8c56-44faeca41695)
